### DISCUSSION TITLE
Is Photoswipe still working?

### DISCUSSION THREAD

**Discussion by shaform** [xianmin/hugo-theme-jane#410] [Q&A] [answered]
While testing out V3, I found that jQuery was removed in https://github.com/xianmin/hugo-theme-jane/commit/d6995a31de3c0317e7bb44a8f5129cbc7bca995a. However, there are some comments that suggest that jQuery must be loaded for photoswipe to work:

https://github.com/xianmin/hugo-theme-jane/blob/1e476ccc61156dc87443f7011cd114a64491b4e1/layouts/partials/photoswipe.html#L7

However, I have not used photoswipe before so cannot verify whether photoswipe is really not working currently. Wondering if anyone has some experience in the photoswipe feature, could you verify to see if photoswipe is still working?

**Comment by xianmin** [comment, marked as answer]
Yes, to reduce dependencies, I removed jQuery. and PhotoSwipe has not been fixed yet. I hope to find a lightweight library similar to PhotoSwipe to replace it. This task is not completed yet. Sorry...
